### PR TITLE
Reorganizing documentation since SimpleFIN will be officially released #2

### PR DESCRIPTION
**Pull request by Jonathan-Fang**
- removed simplefin from experimental tab
- added mention of simplefin in bank-sync
- split bank-sync into gocardless and simplefin
- created dropdown for bank-sync
- proofreaded punctuation & image alignment in gocardless and simplefin guides

Fixes https://github.com/actualbudget/docs/issues/452, https://github.com/actualbudget/docs/pull/453 and fixes https://github.com/actualbudget/docs/issues/454

I'm not sure why https://github.com/actualbudget/docs/issues/458 got closed, so I'm making a new one here.

### CODE REVIEW

File: docs/advanced/bank-sync.md [hunk 1]
```diff
@@ -10,142 +10,13 @@ Here are a couple of considerations to know about before making the decision to
...
+- The integration only works if you are using actual-server.
 
 - The Secrets and Keys are stored in your Actual installed version so it is highly recommended to turn on End to End encryption and create a strong passphrase to encrypt your files.
 
-- You will need to add a config file to your installation
+- You will need to add a config file to your installation.
 
 ## Supported Providers
 
 * GoCardless
```
Comment (matt-fidd) [inline docs/advanced/bank-sync.md:21]: Could you add some indication that GoCardless is for European and UK banks, and that SimpleFin is for US banks?

File: docs/advanced/bank-sync/simplefin.md [hunk 2]
```diff
@@ -1,24 +1,17 @@
-# SimpleFIN Bank Sync
-
-:::warning
-This is an **experimental feature**. That means we’re still working on finishing it. There may be bugs, missing functionality or incomplete documentation, and we may decide to remove the feature in a future release. If you have any feedback, please [open an issue](https://github.com/actualbudget/actual/issues) or post a message in the Discord.
-:::
-:::warning
-All functionality described here may not be available in the latest stable release. Use the `edge` images for the latest implementation.
-:::
+# SimpleFIN Sync Setup
```
Comment (matt-fidd) [inline docs/advanced/bank-sync/simplefin.md:1]: We should be consistent between "GoCardless Setup" and "SimpleFIN Sync Setup". I'd prefer that the latter was changed to "SimpleFIN Setup" to match
```suggestion
# SimpleFIN Setup
```

File: docs/advanced/bank-sync/simplefin.md [hunk 3]
```diff
@@ -28,39 +21,40 @@ All functionality described here may not be available in the latest stable relea
...
 
 5. From the "My Accounts" page, under "Apps", click on "New Connection". Give your connection a name, and click "Create Setup Token"
-   _This is only for you to easily identify it in the SimpleFIN connections overview_
 
-![](/static/img/connecting-your-bank/connecting-your-bank-simplefin-03.png)
+   _This is only for you to easily identify it in the SimpleFIN connections overview._
+
+    ![](/static/img/connecting-your-bank/connecting-your-bank-simplefin-03.png)
 
-5. Save the generated Setup Token someplace safe (one-time use only).
```
Comment (matt-fidd) [inline docs/advanced/bank-sync/simplefin.md:35]: Good catch!

File: docs-sidebar.js [hunk 4]
```diff
@@ -180,7 +180,18 @@ const sidebars = {
...
+              label: 'Connecting Your Bank',
+              link: {
+                type: 'doc',
+                id: 'advanced/bank-sync',
+                    },
+                    items: [
+                    'advanced/bank-sync/gocardless',
+                    'advanced/bank-sync/simplefin',
+                    ],
+            },
```
Comment (matt-fidd) [inline docs-sidebar.js:194]: The formatting here is a little wonky

```suggestion
            {
              type: 'category',
              label: 'Connecting Your Bank',
              link: {
                type: 'doc',
                id: 'advanced/bank-sync',
              },
              items: [
                'advanced/bank-sync/gocardless',
                'advanced/bank-sync/simplefin',
              ],
            },
```

File: docs/advanced/bank-sync.md [hunk 5]
```diff
@@ -10,142 +10,13 @@ Here are a couple of considerations to know about before making the decision to
```
Comment (matt-fidd) [inline docs/advanced/bank-sync.md:10]: Could the note at the top of this page vegarding the minimum version be moved onto the GCL page, and then a new one added to the Simplefin page to show version v24.10.0?

**Review by matt-fidd — CHANGES_REQUESTED** [review]
Thanks for this!
A few quick first pass comments

File: docs/advanced/bank-sync.md (hunk 1 above)
Comment (Jonathan-Fang) [reply docs/advanced/bank-sync.md]: Yep! I've added GoCardless (European Banks) and SimpleFIN (North American Banks) to keep consistent with the web application.
image

Comment (matt-fidd) [reply docs/advanced/bank-sync.md]: Perfect, thank you!

**Review by Jonathan-Fang — COMMENTED** [review]
This is ready for another review!

**Review by matt-fidd — APPROVED** [review]
Looks great, thank you!
I won't merge it until the release goes out so the docs stay in line.

**Pull request merged**